### DISCUSSION TITLE
[Feature] Store multiple formats of the same image in duplicate detection

### DISCUSSION THREAD

**Discussion by Explosion-Scratch** [immich-app/immich#10435] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Duplicate detection frequently turns up results such as this:

<img width="916" alt="image" src="https://github.com/immich-app/immich/assets/61319150/de892de4-5523-4598-baf4-189e0d1e806e">

This is the same image, taken at the same time, but in a different format, auto converted to JPG by the camera. In these cases I would like to keep both the files, which is why I propose a "Merge" button.

The merge button would simply store both images alongside each other and 

### Platform

- [x] Server
- [x] Web
- [x] Mobile

**Comment by bo0tzz** [comment]
This would be accomplished by stacking them: #10148

**Comment by Explosion-Scratch** [comment]
Thanks- Sorry I didn't see the duplicate! I love using immich!
